cmd/kubeadm/app/util/certs/util.go

```diff
@@ -31,12 +32,12 @@ import (
 
 // SetupCertificateAuthority is a utility function for kubeadm testing that creates a
 // CertificateAuthority cert/key pair
-func SetupCertificateAuthority(t *testing.T) (*x509.Certificate, crypto.Signer) {
+func SetupCertificateAuthority() (*x509.Certificate, crypto.Signer) {
```

Comment:
I'd recommend leaving the testing parameter to make it clear this is a test-only method... the package and function name don't indicate it is test-only